leftwm/src/bin/leftwm-worker.rs

```diff
@@ -5,7 +5,6 @@ fn main() {
     leftwm::utils::log::setup_logging();
 
     tracing::info!("leftwm-worker booted!");
-    tracing::debug!("bitte");
```

Comment:
I loved how you could see the struggle that went into getting there :heart_eyes: